### ISSUE TITLE
Validator demo site (and API) should expose version numbers for the frontend, supported schema versions/dates/release, and shex + shacl inputs

### ISSUE THREAD

**Issue by danbri** [google/schemarama#44]
These should also be html visible somewhere, but for now it would be good to know how to go to https://google.github.io/schemarama/demo/ or whatever, and pop up browser devtools and quickly see this info.

We currently have shacl/shex generated from an old release of schema.org, people will ask which release we support.

(should also re-run convertors when Schema project does a release)

/cc @ericprud

**Comment by Gnomus042** [comment]
Added a shapes viewer to the demo, please let me know if this is one of the things you wanted to add :)

For the version numbers, should they be consumed somehow automatically each time schema.org has a release or just kept in a config file we make changes to reflect what was changed in the latest release?

**Comment by danbri** [comment]
Yes - viewing / explaining the rules is really important- thanks!

Version - not sure exactly but … what are the steps to make in this repi whenever schemaorg repo makes and tags a new release? Can we wire that using actions?
